src/ui/Backdrop/Backdrop.ts

```diff
@@ -0,0 +1,74 @@
+import {Component} from '../Base/Component';
+import {ComponentOptions} from '../Base/ComponentOptions';
+import {IComponentBindings} from '../Base/ComponentBindings';
+import {Initialization, IInitializationParameters} from '../Base/Initialization';
+import {IResultsComponentBindings} from '../Base/ResultsComponentBindings';
+import {IQueryResult} from '../../rest/QueryResult';
+
+export interface IBackdropOptions {
+  imageField: string;
+  overlayColor?: string;
+  overlayGradient?: boolean;
+}
+
+/**
+ * This component is used to render an image URL (contained in a result's field)
```

Comment:
Is it me or it's a bit vague/too short.

From reading the code, I can see that it's possible to include other component inside, and that they will be initialized. It should be mentioned here.

Is there any limitation to the type of component I can put inside ?

Any other valuable info you can think of ?